test/rdf-insert-mocha.js

```diff
@@ -0,0 +1,85 @@
+    server.on('request', function(req, res) {
+        var body = [];
+        req.on('data', function(chunk) {
+            body.push(chunk);
+        }).on('end', function() {
+            body = Buffer.concat(body).toString();
+            res.end(body);
+        });
+    });
+    before(function(){
+        server.listen(1337);
+    });
+    after(function(){
+        server.close();
+    });
+    var john = {
+        "@context": "http://json-ld.org/contexts/person.jsonld",
+        "@graph": [{
+            "@id": "http://dbpedia.org/resource/John_Lennon",
+            "name": "John Lennon",
+            "born": "1940-10-09",
+            "spouse": "http://dbpedia.org/resource/Cynthia_Lennon"
+        }]
+    };
+    var sparql = 'INSERT DATA {\n' +
+        '<http://dbpedia.org/resource/John_Lennon> <http://schema.org/birthDate> "1940-10-09"^^<http://www.w3.org/2001/XMLSchema#dateTime> .\n' +
+        '<http://dbpedia.org/resource/John_Lennon> <http://schema.org/spouse> <http://dbpedia.org/resource/Cynthia_Lennon> .\n' +
+        '<http://dbpedia.org/resource/John_Lennon> <http://xmlns.com/foaf/0.1/name> "John Lennon" .\n' +
+        '}\n';
+    it("should POST jsonld as SPARQL INSERT", function() {
+        return test.createNetwork({
+            loadJson: rdfLoad,
+            ntriples: rdfNtriples,
+            request: requestTemplate
+        }).then(function(network){
+            var output = noflo.internalSocket.createSocket();
+            network.processes.request.component.outPorts.output.attach(output);
+            return new Promise(function(done) {
+                output.on('data', done);
+                network.graph.addEdge('loadJson', 'output', 'ntriples', 'input');
+                network.graph.addEdge('ntriples', 'output', 'request', 'input');
+                network.graph.addInitial('POST', 'request', 'method');
+                network.graph.addInitial("http://localhost:1337/", 'request', 'url');
+                network.graph.addInitial({'Content-Type': 'text/turtle'}, 'request', 'headers');
+                network.graph.addInitial('INSERT DATA {\n{{#each tokens}}{{{this}}}{{/each}}\n}', 'request', 'body');
+                network.graph.addInitial(john, 'loadJson', 'input');
+            }).then(function(sparql){
+                return sparql.replace(/\s+/g,'\n').trim();
+            });
+        }).should.become(sparql.replace(/\s+/g,'\n').trim());
+    });
+    it("should POST jsonld as SPARQL INSERT using the default graph", function() {
+        return test.createNetwork({
+            insert: "rdf-components/rdf-insert"
+        }).then(function(network){
+            var output = noflo.internalSocket.createSocket();
+            network.processes.insert.component.outPorts.output.attach(output);
+            return new Promise(function(done) {
+                output.on('data', done);
+                network.graph.addInitial("http://localhost:1337/", 'insert', 'sparql_endpoint');
```

Comment:
Since the port number is being used multiple places, can we put it into a variable so it can be changed if we want to?